src/widgets/SimpleTable.js

```diff
@@ -0,0 +1,187 @@
+/* eslint-disable react/no-unused-prop-types */
+/* eslint-disable react/forbid-prop-types */
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { StyleSheet, FlatList, View, Text } from 'react-native';
+import { ROW_BLUE, APP_FONT_FAMILY, SUSSOL_ORANGE } from '../globalStyles/index';
+import { generateUUID } from '../database';
+
+/**
+ * Simple table component only rendering scalar values in a cell.
+ * Required props
+ * data = [ { a: 1, b: 2}, {a: 3, b: 4}, .. ]
+ * columns = [
+ * { key: 'a', width: 1, title: 'A COLUMN' },
+ * { key: 'b', width: 1, title: 'B COLUMN' } ..
+ * ]
+ * title: Title of the table.
```

Comment:
Bit of a nit: This comment block should follow the format of the rest of the codebase.

Reply:
done 58b5855d83a44af0d710c3b6e03effc6f94cb806